include/mbedtls/mbedtls_config.h

```diff
@@ -2848,15 +2848,11 @@
  *
  * Module:  library/pkcs5.c
  *
- * Requires: MBEDTLS_CIPHER_C and either MBEDTLS_MD_C or MBEDTLS_PSA_CRYPTO_C.
+ * Requires: MBEDTLS_CIPHER_C
+ *           (auto-enables MBEDTLS_MD_C)
```

Comment:
There's a slight discrepancy in this commit: 
it's 'MBEDTLS_PKCS5_C', not 'MBEDTLS_CIPHER_C' that enables 'MBEDTLS_MD_C'.

Reply:
Ah, I see how this can invite confusing. I'll move or reword.